package.json

```diff
@@ -23,6 +23,7 @@
 		"normalize-path": "^3.0.0",
 		"picomatch": "^2.2.2",
 		"reflect-metadata": "^0.1.13",
+		"retire": "^2.2.2",
```

Comment:
We will need a 3pp request if not already filed.

Reply:
Will do.

Reply:
Done. https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qB0000000VB0dIAG/view

Reply:
Approval received.